test/test_cost.py

```diff
@@ -1064,7 +1052,6 @@ def test_qmix_tensordict_keys(self, td_est):
             "action": ("agents", "action"),
             "reward": "reward",
             "done": "done",
-            "terminated": "terminated",
         }
```

Comment:
Why are all these `"terminated"` gone? (not copying the same comment everywhere but I see most of them have disappeared)